read-the-docs-2016-stats.rst

```diff
@@ -0,0 +1,141 @@
+.. is:issue  created:2016-01-01..2017-01-01 
+
+Our community activity is much lower this year,
+which is a multi-year trend that we've been seeing.
+I believe this is in part because of a maturing project and code base,
+and also because we've struggled getting more contributors.
+The code base and project are reasonably hard to set up,
+and we haven't had enough time and focus on onboarding contributors into the project.
+
+I'm not sure what the best path forward here is.
+I've personally onboarded and tried to build many people into ongoing contributors,
+and have gotten discouraged with the lackluster results over the years.
+My hope is that we can eventually make it easier to contribute and start to get more folks helping.
+
+The other path forward is to start using some of our funding to build a paid team who helps with the project.
+This seems like the most likely option given the history state of contributions.
+It would be great if projects could exist under ongoing volunteer maintenance,
+but there is a `plethora of examples`_ in the open source ecosystem of this not being the case.
+
+.. _plethora of examples: http://www.fordfoundation.org/library/reports-and-studies/roads-and-bridges-the-unseen-labor-behind-our-digital-infrastructure
+
+Page Views
+----------
+
+Our stats:
+
+* 252 Million Page Views (+48%)
+* 56 Million Unique Visitors (+47%)
+
+.. From Google Analytics
+
+Page view growth has been steady again this year.
+This is particularly impressive given the scale that we're operating at.
+I'm quite happy to see folks using Read the Docs more and more,
+as well as seeing many other projects using our theme or other concepts that we've pioneered in the general software ecosystem.
+
+We remain one of the `largest sites`_ on the internet.
+
+.. _largest sites: http://www.alexa.com/siteinfo/readthedocs.io
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 53673 projects (+91%)
+* 64585 users (+65%)
+
+Read the Docs has some high profile projects that push a lot of traffic.
+There are however thousands of smaller libraries and projects that fill out that full range of documentation that we host.
+
+We are happy to host documentation for all open source projects,
+and are glad the community finds the service useful.
+
+Funding
+-------
+
+Our hosting costs continue to be sponsored by `Rackspace`_,
+which is fantastically generous of them.
+
+Our largest initiative this year has been our attempt at building `Ethical Advertising`_ into our site.
+The project has been well received,
+which we have been grateful for.
+Funding is a fundamental part of sustainability,
+and we're glad people are supportive of our efforts to get there.
+
+The best outcome of our sustainability efforts is that we're going to be able to hire someone part time to handle support for our users.
+This has always been somewhere that I've felt we have not done our best work,
+and I'm excited we will be able to give it more sustained work.
```

Comment:
The tone here is great, perhaps emulate this in the paragraphs above on contributor status.